docs/components/confirm.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Confirm
+layout: page.jade
+sidebar: true
+collection: docs
+priority: 3
+path: confirm
+section: js
+---
+
+# Confirm
+<p class="lead">
+  A confirm is used to present a question messages to users' actions.
+</p>
+
+## Summary
+1. [Usage](#usage)
+2. [Options](#options)
+3. [Size option](#size-option)
+
+
+## Usage
+The confirm component is an empty container where you can add a message that will be presented to users and receives an answer about something. It has an Ok and Cancel buttons by default
+
+A confirm box is often used when you want to make a confirmation about something with the user.
+
+You can initiate the component as a jQuery plugin:
+```js
+// using any selector
+$('any-selector').on('click', () => $.fn.confirm(callback, options));
+```
+
+Or as a vanilla constructor:
+
+```js
+import Confirm from 'garden/src/js/components/confirm';
+
+new Confirm(callback, options);
+```
+
+The confirm takes a callback in the first argument and the options in the second argument.
+
+Here's an example on how to create a button to open an confirm.
```

Comment:
Aqui seria `a confirm`, mas acho que ficaria ainda melhor algo como `.. a confirmation window | box.` ou `... the component.`